packages/fiori/src/ShellBar.js

```diff
@@ -464,6 +464,7 @@ class ShellBar extends UI5Element {
 	}
 
 	_menuItemPress(event) {
+		this.menuPopover.close();
```

Comment:
The API reference of the "menu-item-click"  event states that the event is:
"Fired, when a menu item is activated Note: You can prevent closing of oveflow popover by calling event.preventDefault()."

I think that the prevention does not seem to be supported, you will have to listen for the result if "fireEvent", to check if the event has been prevented and then decide whether to close the overflow or keep it open. Can you check that? @dobrinyonkov